xf-smoke/src/main/java/com/adobe/cq/cloud/testing/it/xf/smoke/Constants.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Adobe Systems Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.adobe.cq.cloud.testing.it.xf.smoke;
+
+final class Constants {
```

Comment:
I think this class doesn't bring much value, each test class can define its own values and it's just a coincidence the values are the same.

Reply:
I've removed the `Constants` class. It turned out that the content package is not even required for `XfSmokeIT`, so I'm fine with the suggested change (no duplication in this case).